client/src/Pages/Registratie.tsx

```diff
@@ -104,6 +134,9 @@ class Registratie extends React.Component<IProps,IState>{
         const value = target.type === 'checkbox' ? target.checked : target.value;
         const name = target.name;
 
+        // Kijk of het ingevulde email uniek is of niet.
+        this.checkEmail();
+
```

Comment:
E-mail check moet alleen worden uitgevoerd als je de e-mail veranderd. E-mail check werkt ook niet als je een ander vak veranderd. Dan geeft hij aan dat het wel kan

Reply:
Dit commentaar zal meegenomen worden in de volgende sprint.